### DISCUSSION TITLE
Ability to hide calendars

### DISCUSSION THREAD

**Discussion by iprabhakar123** [davish/obsidian-full-calendar#364] [Feature Requests]
I have a lot of calendars for different purposes; it would be very useful to hide certain calednars to reduce visual clutter. The current workaround is to go to settings and delete the calendar and load it when necessary but that has too much friction

**Comment by AlphabetsAlphabets** [comment]
Hi, @davish are there any updates on this?

**Comment by YouFoundJK** [comment]
This is now possible in my fork via [workspaces](https://youfoundjk.github.io/plugin-full-calendar/views/workspaces/).
